test/C/HelloWorld.lf

```diff
@@ -1,6 +1,7 @@
 target C {
     tracing: true,
-    logging: error
+    logging: error,
+    docker: {FROM: "alpine:latest"}
```

Comment:
Shouldn't this be `docker-options`? The value of `docker` should be a `boolean` according to TargetProperty.java.

That said, we could also make this a type union of "boolean" or "docker options", in which case we can do with a single target property `docker` (one would not specify any docker options if they weren't interested in using it, and `true` would be a sensible value to use when a default configuration is desired...

Reply:
This is done. There is now just one docker target option that can be a boolean or a list of options.  We should do the same for clock-sync options.